### ISSUE TITLE
Doesn't work with multiple Field > BasicLit

### ISSUE THREAD

**Issue by cometkim** [fabien0102/go2dts#9]
Hey @fabien0102 . This would be a great tool!

I got errors when I try to use this on https://github.com/mattermost/mattermost-server/tree/master/model

```
TypeError: /\t\w* *([a-zA-Z_0-9.*\[\]]+) *`[a-zA-Z:",0-9= ]*json:"(\w+)/.exec is not a function or its return value is not iterable
    at parseParameter (/home/comet/.config/yarn/global/node_modules/go2dts/src/index.js:120:71)
    at Array.map (<anonymous>)
    at readdirSync.filter.forEach.fileName (/home/comet/.config/yarn/global/node_modules/go2dts/src/index.js:72:14)
    at Array.forEach (<anonymous>)
    at srcFolders.forEach.srcFolder (/home/comet/.config/yarn/global/node_modules/go2dts/src/index.js:18:8)
    at Array.forEach (<anonymous>)
    at go2dts (/home/comet/.config/yarn/global/node_modules/go2dts/src/index.js:15:14)
    at Command.program.version.usage.action.args (/home/comet/.config/yarn/global/node_modules/go2dts/bin/go2dts.js:20:5)
    at Command.listener (/home/comet/.config/yarn/global/node_modules/commander/index.js:315:8)
    at Command.emit (events.js:198:13)
	HasSyncables bool        `db:"-" json:"has_syncables"` can't be parsed
TypeError: /\t\w* *([a-zA-Z_0-9.*\[\]]+) *`[a-zA-Z:",0-9= ]*json:"(\w+)/.exec is not a function or its return value is not iterable
    at parseParameter (/home/comet/.config/yarn/global/node_modules/go2dts/src/index.js:120:71)
    at Array.map (<anonymous>)
    at readdirSync.filter.forEach.fileName (/home/comet/.config/yarn/global/node_modules/go2dts/src/index.js:72:14)
    at Array.forEach (<anonymous>)
    at srcFolders.forEach.srcFolder (/home/comet/.config/yarn/global/node_modules/go2dts/src/index.js:18:8)
    at Array.forEach (<anonymous>)
    at go2dts (/home/comet/.config/yarn/global/node_modules/go2dts/src/index.js:15:14)
    at Command.program.version.usage.action.args (/home/comet/.config/yarn/global/node_modules/go2dts/bin/go2dts.js:20:5)
    at Command.listener (/home/comet/.config/yarn/global/node_modules/commander/index.js:315:8)
    at Command.emit (events.js:198:13)
	MemberCount  *int        `db:"-" json:"member_count,omitempty"` can't be parsed
/home/comet/.config/yarn/global/node_modules/go2dts/src/index.js:78
            const values = details.map(({ type }) => type).join(",");
                                          ^

TypeError: Cannot destructure property `type` of 'undefined' or 'null'.
    at details.map (/home/comet/.config/yarn/global/node_modules/go2dts/src/index.js:78:40)
    at Array.map (<anonymous>)
    at readdirSync.filter.forEach.fileName (/home/comet/.config/yarn/global/node_modules/go2dts/src/index.js:78:36)
    at Array.forEach (<anonymous>)
    at srcFolders.forEach.srcFolder (/home/comet/.config/yarn/global/node_modules/go2dts/src/index.js:18:8)
    at Array.forEach (<anonymous>)
    at go2dts (/home/comet/.config/yarn/global/node_modules/go2dts/src/index.js:15:14)
    at Command.program.version.usage.action.args (/home/comet/.config/yarn/global/node_modules/go2dts/bin/go2dts.js:20:5)
    at Command.listener (/home/comet/.config/yarn/global/node_modules/commander/index.js:315:8)
    at Command.emit (events.js:198:13)
```

I previously tried the same with go/parser, it might be difficult to achieve with the only regex... anyway. Good luck to you!

**Comment by fabien0102** [comment]
For sure, regex become a bit too hard at some point 😁, but was working quite well for the scope this little project ^^ We actually moved to a real open-api specs with a custom generator (https://github.com/contiamo/restful-react/blob/master/src/scripts/import-open-api.ts), so we can have types for any backends (not only golang).

If you need this tool, I can try to fix it, it's always funny to play with regex, and your usecase don't seam totally impossible ^^

**Comment by fabien0102** [comment]
@cometkim Should be fix in `v0.3.4` (at least for this bug) let me know if you find something else :) (even if I know that the inheritance is not working 😬, with a bit of luck, you don't have any in this project 🤞 )

**Comment by fabien0102** [comment]
I couldn't resist, I've just fixed the inheritance 😇 Everything is on `v0.4.0` 😉 Enjoy!
